content/javascript/concepts/objects/objects.md

```diff
@@ -96,3 +96,64 @@ This will output:
 ```shell
 Hi, I'm Elizabeth Harmon.
 ```
+
+## Object Classes
+
+Classes are essentially boiler plate object templates. If a car was an object, then a car factory is an object class.
+A class can be constructed with the following notation. The person object example will be used. **Note: annonymous functions can't be used in Classes**
+
+```js
+class Person {
+  greeting() {
+    return `Hi, I'm ${this.firstName} ${this.lastName}.`;
+  }
+}
+```
+
+To use a class an instance of it needs to be created. To demonstrate how an instance of a class is an object, it's properties will be defined using dot notation.
+
+```js
+const person = new Person();
+person.firstName='Elizabeth'
+person.lastName:'Harmon'
+person.age=22
+person.eyeColor='Hazel'
+//If you run console.log(person), you'll get the following:
```

Comment:
Per the content standards, let's avoid first- and second-person pronouns like "we" and "you", if possible, since this content is just a reference entry. 
```suggestion
console.log(person)
/*  Output: Person {
  firstName: 'Elizabeth',
  lastName: 'Harmon',
  age: 22,
  eyeColor: 'Hazel'
} */
```